docs/content/doc/features/comparison.en-us.md

```diff
@@ -60,7 +60,7 @@ _Symbols used in table:_
 | Git LFS 2.0 | ✓ | ✘ | ✓ | ✓ | ✓ | ⁄ | ✓ |
 | Group Milestones | ✘ | ✘ | ✘ | ✓ | ✓ | ✘ | ✘ |
 | Granular user roles (Code, Issues, Wiki etc) | ✓ | ✘ | ✘ | ✓ | ✓ | ✘ | ✘ |
-| Verified Committer | ✘ | ✘ | ? | ✓ | ✓ | ✓ | ✘ |
+| Verified Committer | / | ✘ | ? | ✓ | ✓ | ✓ | ✘ |
```

Comment:
```suggestion
| Verified Committer | ⁄ | ✘ | ? | ✓ | ✓ | ✓ | ✘ |
```

Slash here seems to be some fancy unicode char.

Reply:
done